tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_withdrawal_request.py

```diff
@@ -931,6 +945,13 @@ def run_withdrawal_request_processing(
         # No-op
         assert pre_state == state
     else:
+        validator_index = get_validator_index_by_pubkey(
+            state, withdrawal_request.validator_pubkey
+        )
```

Comment:
For consistency, we should use `pre_state` here.
```suggestion
        validator_index = get_validator_index_by_pubkey(
            pre_state, withdrawal_request.validator_pubkey
        )
```